python/01_setup_infrastructure.py

```diff
@@ -1,158 +1,112 @@
 import fusion
+import json
 import os
 from fusion.rest import ApiException
 from pprint import pprint
 from utils import wait_operation_succeeded
 
-# Setup Config
-config = fusion.Configuration()
-config.issuer_id = os.getenv("API_CLIENT")
-config.private_key_file = os.getenv("PRIV_KEY_FILE")
+def setup_infrastructure():
+    # Setup Config
+    config = fusion.Configuration()
+    config.issuer_id = os.getenv("API_CLIENT")
+    config.private_key_file = os.getenv("PRIV_KEY_FILE")
 
-client = fusion.ApiClient(config)
-r = fusion.RegionsApi(api_client=client)
-az = fusion.AvailabilityZonesApi(api_client=client)
-se = fusion.StorageEndpointsApi(api_client=client)
-a = fusion.ArraysApi(api_client=client)
-nig = fusion.NetworkInterfaceGroupsApi(api_client=client)
-ni = fusion.NetworkInterfacesApi(api_client=client)
+    client = fusion.ApiClient(config)
+    r = fusion.RegionsApi(api_client=client)
+    az = fusion.AvailabilityZonesApi(api_client=client)
+    se = fusion.StorageEndpointsApi(api_client=client)
+    a = fusion.ArraysApi(api_client=client)
+    nig = fusion.NetworkInterfaceGroupsApi(api_client=client)
+    ni = fusion.NetworkInterfacesApi(api_client=client)
 
-region = "region1"
-availability_zone = "az1"
+    # Load configuration
+    with open("config/infrastructure.json") as file:
```

Comment:
Btw this means that we need to execute the python script from within the python directory.

There are usually 2 ways we can do to allow users to execute the python scripts from anywhere:
1. Take path to json from the command line
2. Make json path relative to where the python script is, instead of where the user's cwd

Reply:
Another suggestion is to use yaml instead of json. yaml allows you to add comments in the config itself